.github/workflows/mac.yml

```diff
@@ -45,12 +45,10 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
-          architecture: x64
```

Comment:
Can we remove the architecture line above as well?

Reply:
yes